python/rainbow/simulators/prox_soft_bodies/solver.py

```diff
@@ -1,6 +1,8 @@
 import rainbow.simulators.prox_soft_bodies.mechanics as MECH
 import rainbow.simulators.prox_soft_bodies.collision_detection as CD
-import rainbow.simulators.prox_rigid_bodies.gauss_seidel as GS
```

Comment:
So the old version got killed?

I would like to have a fall-back option to the vanilla version of GS? Should not be hard to have 3 cases in the if-statements, "gauss-seidel", "parallel gauss-seidel", and "parallel Jacobi"?